content/js/getting-started.md

```diff
@@ -0,0 +1,46 @@
+<a name="sign-up-portal"></a>
+## Signing up at Portal
+
+To start using the Skygear JS SDK, you need to register your account and
+application at the [Skygear Developer Portal](https://portal.skygear.io/signup).
+You will be able to monitor your Skygear server and acquire information, such
+as your `Server EndPoint` and `API Key`, on the portal page.
+
+<a name="about-js-sdk"></a>
+## About the SDK
+
+Skygear provides a complete and open-source backend solution for your
+web applications. Our aim is to free front end developers from worrying about
+writing backend code or maintaining database.
+
+The SDK fully abstracts the way you interact with server so that you no longer
+need to deal with complicated AJAX requests or socket connections. The SDK
+talks to your Skygear server directly. With simple API calls and elegant
+Promise syntax, you can achieve almost anything in your web app:
+- Signing up a new user
+- Logging in an existing user
+- Saving and retrieving data
+- Querying data with complicated conditions
+- Setting access control for different users
+- Managing relationship between users
+- Sending and receiving instant message/data via Publish-Subscribe service
+- Working with geolocation data
+- Uploading images and files
+
+The SDK is written in [Babel 6 ES 2015](https://babeljs.io/), available on
```

Comment:
I believe the SDK is available not as ES2015 (ES6) but ES5 because it has already been compiled into ES5 by Babel. Or simply not to talk about Babel/ES2015?